draft-ietf-rtcweb-jsep.xml

```diff
@@ -1097,6 +1097,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           simulcast as described in
           <xref target="sec.simulcast" />).</t>
         </section>
+
+        <section title="stopTransceiver" anchor="sec.transceiver-stop">
```

Comment:
To avoid confusion, rather than jamming these on PeerConnection, I think we probably want to make a separate section for RtpTransceivers, e.g.

    4.  Interface
         4.1.  PeerConnection
           4.1.1.  Constructor
           4.1.2.  addTrack
           4.1.3.  addTransceiver
         4.2.  RtpTransceiver
           4.2.1. setDirection 
           4.2.2. stop
           4.2.3. isStopped

(we can probably remove the empty 'Methods' section)
   
...